.github/workflows/build.yaml

```diff
@@ -1,15 +1,8 @@
 name: Build firmwares
 
 on:
+  pull_request:
   push:
-    paths:
-      - Dockerfile
-      - .github/workflows/build.yaml
-      - manifests/**/*.yaml
-    branches:
-      - main
-    tags:
-      - '*'
```

Comment:
Did you meant to delete those this time? :sweat_smile:

Reply:
I think just `paths` is enough, as it would let us edit the README? This is just to prevent unnecessary builds I think.